ci/run-benchmarks.sh

```diff
@@ -11,6 +11,7 @@ if [[ $# -ne 0 ]]; then
     exit 1
 fi
 
-bash /root/capture-backtrace.sh test/benchmarks \
+# we taskset to pin the process to a specific set of cores and their HTs to reduce test result variance
+bash /root/capture-backtrace.sh taskset -c 4-5,16-17 test/benchmarks \
```

Comment:
Why `4-5,16-17` and not `0-1,12-13`? Is it arbitrary? In either case, for future reference consider adding a note that this is specific to our gpuc2 server, which uses a Threadripper such and such.

Reply:
It's not completely arbitrary, I wanted to use a node range that's not in the first CCX, but contained in one CCX. I'll add a note.